src/modules/flyout/flyout.component.scss

```diff
@@ -1,44 +1,50 @@
-@import "../../scss/mixins";
+@import '../../scss/mixins';
 
 .sky-flyout {
-  position:fixed;
-  right:0;
-  top:0;
-  bottom:0;
-  height:100%;
-  background-color:#fff;
-  border-left:6px solid $sky-highlight-color-info;
-  box-shadow:-3px 0 3px 0 rgba(0,0,0,.3);
-
-  width:100%;
-
-  z-index:1001; // omnibar is 1000, help is 9999
-  overflow-y:scroll;
+  @include sky-shadow;
+  position: fixed;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: 100%;
+  background-color: #fff;
+  border-left: 6px solid $sky-highlight-color-info;
+  z-index: $sky-flyout-z-index;
```

Comment:
I'm concerned about this. The sky-flyout-z-index is 999 but omnibar is 1000. I believe the choice to do 1001 was to ensure the flyout was higher than the omnibar. @Blackbaud-AlexKingman am I right there?

Reply:
Disregard. I can't read.....